glib/Gir_GObject.toml

```diff
@@ -103,6 +103,44 @@ concurrency = "send+sync"
     name = "source"
     generate = ["notify"]
 
+[[object]]
+name = "GObject.SignalGroup"
+status = "generate"
+final_type = true
+concurrency = "send+sync"
+    [[object.function]]
+    name = "connect"
+    ignore = true
+    [[object.function]]
+    name = "connect_after"
+    ignore = true
+    [[object.function]]
+    name = "connect_data"
+    ignore = true
+    [[object.function]]
+    name = "connect_object"
+    ignore = true
+    [[object.function]]
+    name = "connect_swapped"
+    ignore = true
+    [[object.function]]
+    name = "connect_closure"
+    rename = "connect"
+    manual = true
+    [[object.function]]
+    name = "dup_target"
+    rename = "target"
+    [[object.property]]
+    name = "target"
+    # notify signal is racy, use connect_bind/connect_unbind instead
```

Comment:
In what sense?

Reply:
The `bind` signal holds the internal mutex during emission and also emits the target as an argument to the handler, so is safe to reference the target from another thread. The docs mention it but don't elaborate: https://docs.gtk.org/gobject/signal.SignalGroup.bind.html